crates/client/src/client/schema/block.rs

```diff
@@ -84,6 +111,16 @@ pub struct Block {
     pub transactions: Vec<TransactionIdFragment>,
 }
 
+/// Block with full transaction data
+#[derive(cynic::QueryFragment, Debug)]
+#[cynic(schema_path = "./assets/schema.sdl", graphql_type = "Block")]
+pub struct FullBlock {
```

Comment:
Do we really want to include it into the main `fuel-core-client`? It is available via [`fuel-core-client-ext`](https://github.com/FuelLabs/fuel-core-client-ext) and not used by the DSK

Reply:
Oh I didn't realize the -ext was a thing. I'll remove these.